SQLitePCLRaw.bundle_sqlite3mc/SQLitePCLRaw.bundle_sqlite3mc.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>SQLitePCLRaw.batteries_v2</AssemblyName>
+    <RootNamespace>SQLitePCL</RootNamespace>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <Description>A SQLitePCLRaw bundle for SQLite3 Multiple Ciphers--an encryption extension for SQLite with support for multiple ciphers.</Description>
```

Comment:
We may want to make it clearer that this is not part of the SQLitePCLRaw project.

> ❗Notice: This library is maintained separately from the SQLitePCLRaw project.

Reply:
> We may want to make it clearer that this is not part of the SQLitePCLRaw project.

I fully agree. Such anote should be added to the description.